package.json

```diff
@@ -42,10 +47,16 @@
     "varint": "^5.0.0"
   },
   "devDependencies": {
-    "aegir": "^12.3.0",
+    "aegir": "git+https://github.com/ipfs/aegir#flow",
+    "babel-cli": "6.26.0",
+    "babel-core": "6.26.0",
+    "babel-loader": "7.1.2",
+    "babel-preset-flow-node": "2.0.1",
     "buffer-equal": "1.0.0",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
+    "flow-bin": "^0.66.0",
```

Comment:
I would recommend upgrading to flow@0.69.0 at this point and probably it would probably make more sense to have flow-bin bundled with aegir.